.gitignore

```diff
@@ -381,3 +381,4 @@ FodyWeavers.xsd
 .DS_Store
 Intel_SighPP_CMake/.DS_Store
 Intel_SighPP_CMake/Intel_SighPP_CMake/.DS_Store
+Intel_SighPP_CMake/Intel_SighPP_CMake/.DS_Store
```

Comment:
the line above with .DS_Store shoud be enough, the files that are already tracked need to be removed from tracking by git to be removed 😄